README.md

```diff
@@ -978,6 +978,9 @@ arXiv:1907.07469, 2019.
 - <a name="Katz12iscas"></a>Katz, M. L., Nikolic, K., Delbruck, T. (2012),  
 *[Live demonstration: Behavioural emulation of event-based vision sensors](https://doi.org/10.1109/ISCAS.2012.6272143)*,  
 IEEE Int. Symp. Circuits and Systems (ISCAS), 2012, pp. 736-740. [PDF](https://drive.google.com/open?id=1asyMIQIAtsl3_KqgfPuZagr_gjKZhbYm)
+- <a name="kaiser2016towards"></a>Kaiser, J., Tieck, J. C. V., Hubschneider, C., Wolf, P., Weber, M., Hoff, M., Friedrich., A., Wojtasik, K., Roennau, A., Kohlhaas, R., Dillmann, R., Zoellner, M. (2016),  
+*[Towards a framework for end-to-end control of a simulated vehicle with spiking neural networks](https://doi.org/10.1109/SIMPAR.2016.7862386)*,  
+IEEE Int. Conf. on Simulation, Modeling, and Programming for Autonomous Robots (SIMPAR), 2016, pp. 127-134. [PDF](https://www.researchgate.net/api/literature/privateDownload?publicationUid=ZdBlT7Qqmb6_ZZObCvMnubw2q8KDjicjFRfic0lhKDRqWKOJ7d9mRlrSHFzIoUOs7Q&linkId=39iRA3V82odqjV_bWvx3usAc40GmnH5-_VVPDSdbyVn-bjigU7kluQkOVWNKTfla0cU5_HBO7gtbWL8o8IQeyA), [Gazebo DVS plugin](https://github.com/HBPNeurorobotics/gazebo_dvs_plugin)
```

Comment:
The researchgate link does not work (at least for me). Do you have another link? Perhaps an open access repository at your institution?